Traceback (most recent call last):
  File "/Users/wesleyhoang/Desktop/code/cymate-template/web-template/src/main/python/company-template/tests/run_tests.py", line 61, in <module>
    run_tests()
  File "/Users/wesleyhoang/Desktop/code/cymate-template/web-template/src/main/python/company-template/tests/run_tests.py", line 51, in run_tests
    pytest.main(args)
  File "/Users/wesleyhoang/Desktop/code/cymate-template/web-template/venv/lib/python3.10/site-packages/_pytest/config/__init__.py", line 148, in main
    config = _prepareconfig(args, plugins)
  File "/Users/wesleyhoang/Desktop/code/cymate-template/web-template/venv/lib/python3.10/site-packages/_pytest/config/__init__.py", line 329, in _prepareconfig
    config = pluginmanager.hook.pytest_cmdline_parse(
  File "/Users/wesleyhoang/Desktop/code/cymate-template/web-template/venv/lib/python3.10/site-packages/pluggy/_hooks.py", line 265, in __call__
    return self._hookexec(self.name, self.get_hookimpls(), kwargs, firstresult)
  File "/Users/wesleyhoang/Desktop/code/cymate-template/web-template/venv/lib/python3.10/site-packages/pluggy/_manager.py", line 80, in _hookexec
    return self._inner_hookexec(hook_name, methods, kwargs, firstresult)
  File "/Users/wesleyhoang/Desktop/code/cymate-template/web-template/venv/lib/python3.10/site-packages/pluggy/_callers.py", line 55, in _multicall
    gen.send(outcome)
  File "/Users/wesleyhoang/Desktop/code/cymate-template/web-template/venv/lib/python3.10/site-packages/_pytest/helpconfig.py", line 103, in pytest_cmdline_parse
    config: Config = outcome.get_result()
  File "/Users/wesleyhoang/Desktop/code/cymate-template/web-template/venv/lib/python3.10/site-packages/pluggy/_result.py", line 60, in get_result
    raise ex[1].with_traceback(ex[2])
  File "/Users/wesleyhoang/Desktop/code/cymate-template/web-template/venv/lib/python3.10/site-packages/pluggy/_callers.py", line 39, in _multicall
    res = hook_impl.function(*args)
  File "/Users/wesleyhoang/Desktop/code/cymate-template/web-template/venv/lib/python3.10/site-packages/_pytest/config/__init__.py", line 1058, in pytest_cmdline_parse
    self.parse(args)
  File "/Users/wesleyhoang/Desktop/code/cymate-template/web-template/venv/lib/python3.10/site-packages/_pytest/config/__init__.py", line 1346, in parse
    self._preparse(args, addopts=addopts)
  File "/Users/wesleyhoang/Desktop/code/cymate-template/web-template/venv/lib/python3.10/site-packages/_pytest/config/__init__.py", line 1248, in _preparse
    self.hook.pytest_load_initial_conftests(
  File "/Users/wesleyhoang/Desktop/code/cymate-template/web-template/venv/lib/python3.10/site-packages/pluggy/_hooks.py", line 265, in __call__
    return self._hookexec(self.name, self.get_hookimpls(), kwargs, firstresult)
  File "/Users/wesleyhoang/Desktop/code/cymate-template/web-template/venv/lib/python3.10/site-packages/pluggy/_manager.py", line 80, in _hookexec
    return self._inner_hookexec(hook_name, methods, kwargs, firstresult)
  File "/Users/wesleyhoang/Desktop/code/cymate-template/web-template/venv/lib/python3.10/site-packages/pluggy/_callers.py", line 60, in _multicall
    return outcome.get_result()
  File "/Users/wesleyhoang/Desktop/code/cymate-template/web-template/venv/lib/python3.10/site-packages/pluggy/_result.py", line 60, in get_result
    raise ex[1].with_traceback(ex[2])
  File "/Users/wesleyhoang/Desktop/code/cymate-template/web-template/venv/lib/python3.10/site-packages/pluggy/_callers.py", line 39, in _multicall
    res = hook_impl.function(*args)
  File "/Users/wesleyhoang/Desktop/code/cymate-template/web-template/venv/lib/python3.10/site-packages/_pytest/config/__init__.py", line 1125, in pytest_load_initial_conftests
    self.pluginmanager._set_initial_conftests(
  File "/Users/wesleyhoang/Desktop/code/cymate-template/web-template/venv/lib/python3.10/site-packages/_pytest/config/__init__.py", line 557, in _set_initial_conftests
    self._try_load_conftest(anchor, namespace.importmode, rootpath)
  File "/Users/wesleyhoang/Desktop/code/cymate-template/web-template/venv/lib/python3.10/site-packages/_pytest/config/__init__.py", line 574, in _try_load_conftest
    self._getconftestmodules(anchor, importmode, rootpath)
  File "/Users/wesleyhoang/Desktop/code/cymate-template/web-template/venv/lib/python3.10/site-packages/_pytest/config/__init__.py", line 603, in _getconftestmodules
    mod = self._importconftest(conftestpath, importmode, rootpath)
  File "/Users/wesleyhoang/Desktop/code/cymate-template/web-template/venv/lib/python3.10/site-packages/_pytest/config/__init__.py", line 651, in _importconftest
    self.consider_conftest(mod)
  File "/Users/wesleyhoang/Desktop/code/cymate-template/web-template/venv/lib/python3.10/site-packages/_pytest/config/__init__.py", line 732, in consider_conftest
    self.register(conftestmodule, name=conftestmodule.__file__)
  File "/Users/wesleyhoang/Desktop/code/cymate-template/web-template/venv/lib/python3.10/site-packages/_pytest/config/__init__.py", line 489, in register
    ret: Optional[str] = super().register(plugin, name)
  File "/Users/wesleyhoang/Desktop/code/cymate-template/web-template/venv/lib/python3.10/site-packages/pluggy/_manager.py", line 114, in register
    self._verify_hook(hook, hookimpl)
  File "/Users/wesleyhoang/Desktop/code/cymate-template/web-template/venv/lib/python3.10/site-packages/pluggy/_manager.py", line 246, in _verify_hook
    raise PluginValidationError(
pluggy._manager.PluginValidationError: Plugin '/Users/wesleyhoang/Desktop/code/cymate-template/web-template/src/main/python/company-template/tests/conftest.py' for hook 'pytest_runtest_makereport'
hookimpl definition: pytest_runtest_makereport(item, call)
Declared as hookwrapper=True but function is not a generator function
